model/Database.php

```diff
@@ -359,6 +364,33 @@ function getUserByEmail($user_email)
         return $statement->fetch(PDO::FETCH_ASSOC);
     }
 
+    /**
+     * Get id and email for all admins
+     *
+     * @return array admin id and email data
+     * @author Keller Flint
+     */
+    function getAdmins()
+    {
+        $sql = "SELECT user_id, user_email FROM tutors.User WHERE user_is_admin = 1";
+        $statement = $this->_dbh->prepare($sql);
+        $statement->execute();
+        return $statement->fetchAll(PDO::FETCH_ASSOC);
+    }
 
+    /**
+     * Add a new admin
+     *
+     * @param string $email email of the new admin
+     * @return int id of the new admin
+     * @author Keller Flint
+     */
+    function addAdmin($email)
+    {
+        $sql = "INSERT INTO User VALUES (DEFAULT, ?, '1234', b'1')";
```

Comment:
Just for future reference you can pass in a number without quotes. This way you won't have to convert it into binary since any number is already in binary form. Its just a decimal view for us.